main.go

```diff
@@ -25,19 +25,45 @@ func main() {
 	})
 }
 
+func normalizeInput(input string) (string, error) {
```

Comment:
This function makes sure that the content can be updated by making sure yaml from kubernetes and hand-edited YAML is compared by terraform as the same.